File: 02 FormatString/c#/FormatString.cs
var language = "C#";
var price = 24.99F;
byte quotas = 20;

var concatenation = "Course: " + language + " Programming Fundamentals \n" + "Price: $" + price + " \n" + "Quotas: " + quotas + " \n";

var interpolation = $"Course: {language} Programming Fundamentals \nPrice: ${price} \nQuotas: {quotas}";

var format = string.Format("Course: {0} Programming Fundamentals \nPrice: ${1} \nQuotas: {2}", language, price, quotas);

var literalString = @"
Course: C# Programming Fundamentals 
Price: $24.99 
Quotas: 20 

";

File: 02 FormatString/ts/formatString.ts
const language = "C#";
const price = 24.99;
const quotas = 20;

const concatenation =
  "Course: " +
  language +
  " Programming Fundamentals \n" +
  "Price: $" +
  price +
  " \n" +
  "Quotas: " +
  quotas +
  " \n";

const interpolation = `Course: ${language} Programming Fundamentals \nPrice: $${price} \nQuotas: ${quotas}`;
